Automattic/wp-calypso
# Domains: Incoming Domain Transfers

Implementing i1 for incoming domain transfers. It's still rough on the edges, but a working flow w/o the CSS.

As we prepare for i2, we should merge this and iterate.

Changes:
- Add a different component for transferring in when domain taken
- Add paths for transfer to controller
- Add checkout verification for transfer in
- Display transfer ins in the domain management list
- Add notices for transfer ins
- Add checkout screen for transfer in

Test:
- Make sure Domains -> Add still shows `Map It` option, when feature gate is `false`
- Make sure Domains -> Add shows `Use your own domain` otherwise
